### DISCUSSION TITLE
Incoming Reward not visible in wallet

### DISCUSSION THREAD

**Discussion by byozturk17** [Chia-Network/chia-blockchain#6548] [Support]
Hello there,

345147 The reward given in this block does not appear in my wallet. Can someone explain what's going on?

Wallet : xch1u6778jkymeuvrz5mlhl88kckty62aq3klkhk0matqngm6mtq5hrsvp8lyl

![3](https://user-images.githubusercontent.com/85479944/120976949-7bfcc500-c77b-11eb-9047-4ab3219de9f1.PNG)
![1](https://user-images.githubusercontent.com/85479944/120976942-7acb9800-c77b-11eb-908c-1ce70a1ed5b6.PNG)
![2](https://user-images.githubusercontent.com/85479944/120976948-7b642e80-c77b-11eb-8401-571e50e942df.PNG)

**Comment by byozturk17** [comment]
Hello

345147 block appears to have received the reward, but 386926 block section is present in the section removed when the details are examined, it says that the Sent reward has been removed.

My wallet code: xch1u6778jkymeuvrz5mlhl88kckty62aq3klkhk0matqngm6mtq5hrsvp8lyl

Check the reward block from: https://www.chiaexplorer.com/blockchain/block/0xd9378e25de7cc14122e98ed5ccde8f0d46cd8476c891c52f26a59cb48b265a24

Check the block indicating that it has been removed:
https://www.chiaexplorer.com/blockchain/block/0xd95dd9828bfd4544fdad0a1148d936645912a132a267b2ecf46b12b47a23c928

I'm waiting for your help and your return.
Good work.

**Comment by loppefaaret** [comment]
https://www.chiastatus.com/addresses/xch1u6778jkymeuvrz5mlhl88kckty62aq3klkhk0matqngm6mtq5hrsvp8lyl
The address farmed a block, it send off 2 XCH, and then received what looks like a faucet payout
Resyncing a wallet does not show the outgoing transactions, but the amount available is display correctly

**Reply by byozturk17** [reply]
Hello there,

The wallet also does not display XCH. I did not transfer anywhere. What should I do?

**Reply by loppefaaret** [reply]
then your key is most likely compromised, stolen, you should figure out how that happened - if you can't, i would recommend you reinstall your OS, and imidatly after generate a new 24 mnemonic key, and follow the wiki instructions for cold wallet farming:
https://github.com/Chia-Network/chia-blockchain/wiki/Chia-Keys-Management

**Reply by byozturk17** [reply]
Hello there,

It seems to have been deleted in Chia Explorer. There are no transactions in the history section. I did not share the 24-word keyword with anyone. My computer is safe.

![4](https://user-images.githubusercontent.com/85479944/121036918-b5532600-c7b7-11eb-8ab1-62a618b739eb.PNG)
![5](https://user-images.githubusercontent.com/85479944/121037469-2561ac00-c7b8-11eb-90a3-bc16709ad6f7.PNG)

**Reply by loppefaaret** [reply]
you computer might be, but if you did not send out 6 XCH on block 374029 - then someone else has control of your wallet, and your 24 words mnemonic is compromised!

**Comment by aural227** [comment]
Sorry, can't help you, just want to say how amazing it is that one plot can won one block.

**Reply by leonidcan** [reply]
How is it even possible????

**Reply by byozturk17** [reply]
Hello there,

Sorry for my late reply. But of course I didn't win with a PLOT. I just wanted to show you one.

**Reply by byozturk17** [reply]
You can also see many blocks. Removal is available. Chia officials should take care of this.

**Comment by BrandtH22** [comment]
@byozturk17 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
